docs/commands/oc-skills.md

```
@@ -0,0 +1,28 @@
+# oc skills
+>[!NOTE|label:Description]
+> #### Manage Xano skills for AI agents (database optimization, security, best practices).
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h4

(MD001, heading-increment)